Recipes/Recipes.DataAccess/Entities/Relationships/TagRecipeEntity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Recipes.DataAccess.Entities.Relationships
+{
```

Comment:
Minor nitpick but most of the project uses slightly different formatting for namespaces (if you add a semicolon after the namespace name one layer of indentation is removed)

Reply:
fixed up the formatting on (hopefully) all classes